.env.example

```
@@ -26,20 +26,23 @@ BOT_GITHUB_TOKEN=1a2b3c4d5e6f7g8g9i0j
 # Feature: authentication
 # Mocked: Unnecessary (any value can be used)
-# Technically we have a fallback in development so this doesn't even need to be set
+# Any value can be used locally, but it must be set (we fail fast when missing)
 
 SESSION_SECRET=anything_works_here
 MAGIC_LINK_SECRET=whatever_stuff
 
 # Feature: basically everything
 # Mocked: No, must run sqlite locally
 DATABASE_FILENAME="sqlite.db"
+DATABASE_PATH="./prisma/sqlite.db"
```

dotenv-linter (4.0.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)